libminifi/test/TestBase.h

```diff
@@ -200,7 +199,7 @@ class LogTestController {
     }
     my_properties_->set("logger.root", "ERROR,ostream");
     my_properties_->set("logger." + core::getClassName<LogTestController>(), "INFO");
-    my_properties_->set("logger." + core::getClassName<logging::LoggerConfiguration>(), "DEBUG");
+    my_properties_->set("logger." + core::getClassName<logging::LoggerConfiguration>(), "INFO");
```

Comment:
This was added for hiding the `[Component] got sinks from ...`, flooding the test logs. If someone finds those log lines helpful, I am happy to restore this.

Reply:
This brakes C2JstackTest. Once  #953 is merged, rebasing on it should fix this issue.

Reply:
It is no longer an issue.